fastify.js

```diff
@@ -298,16 +285,27 @@ function build (options) {
     }
 
     req.id = genReqId(req)
-    handleTrustProxy(req)
-    req.hostname = req.hostname || req.headers['host']
+
+    var hostname = req.headers['host']
+    var ip = req.connection.remoteAddress
+    var ips
```

Comment:
Can you add a test that checks that `ips`  is `undefined`?

Reply:
If you meant creating a test case for undefined `ips`, currently there are some use cases being tested in [trust-proxy.test.js](https://github.com/fastify/fastify/blob/master/test/trust-proxy.test.js#L39) Is this what you meant or am I way off? Thanks!